.eslintrc

```diff
@@ -21,8 +31,7 @@
   },
   "ignorePatterns": [
     "test/integration/",
-    "dist",
-    "examples"
```

Comment:
I enabled eslint on examples folder, hence the number of changed files on this PR.